VideoLocker/src/main/java/org/edx/mobile/util/FileUtil.java

```diff
@@ -0,0 +1,39 @@
+package org.edx.mobile.util;
+
+import android.content.Context;
+
+import java.io.BufferedInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Created by miankhalid on 8/3/15.
+ */
+public class FileUtil {
+    /**
+     * Returns the text of a file as a String object
+     * @param context   current context
+     * @param fileName  name of the file to load from assets folder
+     * @return  text content of the file
+     */
+    public static String loadTextFileFromAssets(Context context, String fileName) {
+        String fileContent;
+        try {
+            InputStream iStream = context.getAssets().open(fileName);
+            BufferedInputStream bInputStream = new BufferedInputStream(iStream);
+            ByteArrayOutputStream bOutputStream = new ByteArrayOutputStream();
+            int result = bInputStream.read();
+            while(result != -1) {
+                byte b = (byte)result;
+                bOutputStream.write(b);
+                result = bInputStream.read();
+            }
+            fileContent = bOutputStream.toString();
+        } catch (IOException ioexception) {
+            ioexception.printStackTrace();
+            return null;
```

Comment:
This should probably throw an exception so that the caller can decide what to do if it fails. The way the code is here, returning ``null`` here would cause the string "null" to be printed into the styles tag. Better to just not print anything if loading this fails.